taxonomy-manager-rest-server/src/test/resources/jwtClaims.json

```diff
@@ -0,0 +1,12 @@
+{
+  "iss": "https://dev.taxman.digirati.com/",
```

Comment:
Can we use something other than `dev.taxman.digirati.com` here for the purpose of running tests please.